fuel-tx/src/transaction/types/utxo_id.rs

```diff
@@ -20,6 +20,7 @@ use rand::{
 #[derive(Debug, Default, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 #[cfg_attr(feature = "typescript", wasm_bindgen::prelude::wasm_bindgen)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "da-compression", derive(fuel_compression::Compact))]
 #[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
 pub struct UtxoId {
```

Comment:
I'm not planning to do that here, since that's separate from the BiggerType -> Key separation I'm truing to achieve in the PR. Maybe I'll add it here if I can refactor like this https://github.com/FuelLabs/fuel-vm/pull/670#discussion_r1502669109

Reply:
This currently lives in #808

Reply:
Merged into this pr